Projects/PaywallTester/Project.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import ProjectDescription
+import ProjectDescriptionHelpers
+
+let allDestinations: Destinations = [
+    .iPhone,
+    .iPad,
+    .mac,
+    .macWithiPadDesign,
+    .macCatalyst,
+    .appleWatch,
+    .appleTv,
+    .appleVision,
+    .appleVisionWithiPadDesign
+]
+
+let allDeploymentTargets: DeploymentTargets = .multiplatform(
+    iOS: "15.0",
+    macOS: "11.0",
+    watchOS: "7.0",
+    tvOS: "14.0"
+)
+
+var additionalFiles: [FileElement] = [
+    .glob(pattern: "../../Global.xcconfig")
+]
+if FileManager.default.fileExists(atPath: "../../Local.xcconfig") {
+    additionalFiles.append(.glob(pattern: "../../Local.xcconfig"))
+}
+
+let project = Project(
+    name: "PaywallTester",
```

Comment:
Are we adding one of these Projects for `PurchaseTester`? I think that's also very useful and would be great to get in before merging this.

Reply:
Nothing, I've found it 😆 . Adding it

Reply:
Just added it :) `tuist generate PurchaseTester`